pkg/build/webhook/webhook.go

```diff
@@ -3,9 +3,12 @@ package webhook
 import (
 	"crypto/hmac"
 	"errors"
+	"net/http"
 	"strings"
 
-	"github.com/openshift/origin/pkg/build/api"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
```

Comment:
mainly the latter.  as a rule i prefer the more explicit "buildapi", it also makes it easier in the future when you go to add another import that might otherwise collide, but you don't have to go change anything we already have.